### DISCUSSION TITLE
China National Day, working on axmol-3.x draft

### DISCUSSION THREAD

**Discussion by halx99** [axmolengine/axmol#2195] [Announcements]
RT, I notice there are a lot of features I  can do, 
for detail, please see: https://github.com/axmolengine/axmol/milestone/20 , and PR: https://github.com/axmolengine/axmol/pull/2387

The axmol-3.x draft working branch should be https://github.com/axmolengine/axmol/tree/wip-v3, and will in alpha state long terms
 
And also I will release axmol-2.2, there are a lot of release notes need to be tidy, thanks all contributers of axmol, you bring so much features and bugfixes to make axmol better and stable.

**Comment by paulocoutinhox** [comment]
What is the replacement of PhysicsBody etc?

**Reply by halx99** [reply]
will implement based on box2d v3 later

**Reply by paulocoutinhox** [reply]
Perfect :)

**Comment by haoqoo** [comment]
yyds

**Reply by BIGCATDOG** [reply]
cool

**Comment by aismann** [comment]
unpinned and closed
